### ISSUE TITLE
New GC service to collect bloom filters

### ISSUE THREAD

**Issue by mniewrzal** [storj/team-metainfo#120]
To run GC from backup database we agreed that we need to split GC process into two parts:

* separate GC process which will connect to DB and will create bloom filters for all storage nodes
* old satellite GC service will be modified to consume bloom filters created by separate process and will send bloom filters to storage nodes

Currently, we think that best way to pass bloom filters between producer (separate GC process) and satellite will be Storj bucket.
new GC bloom filters satellite command:

* will access Metabase DB and will setup Segments Loop
* will use Segments Loop in the same way like it was done on satellite side
* creation date for bloom filters should be taken from backup creation time
* after calculating all bloom filters it will upload them into Storj bucket
* configuration will allow to set: access grant and bucket location
* will pack 1000 bloom filters per one object in a way that satellite side will be able to read bloom filters before download of pack will be finished
   * maybe we should use zip?
* at the end of upload process will upload special object to flag that all bloom filters were uploaded
* all objects will be uploaded with expiration time defined (configurable)
* before starting each bloom filter creation we will check if target bucket location is empty

**Comment by storjBuildBot** [comment]
Change https://review.dev.storj.io/c/storj/storj/+/8315 mentions this issue.

**Comment by storjBuildBot** [comment]
Change https://review.dev.storj.io/c/storj/storj/+/8312 mentions this issue.

**Comment by storjBuildBot** [comment]
Change https://review.dev.storj.io/c/storj/storj/+/8460 mentions this issue.

**Comment by mniewrzal** [comment]
All changes are merged. If we will need any adjustments lets make a new ticket.
